ci-operator/step-registry/ipi/conf/powervc/ipi-conf-powervc-commands.sh

```
@@ -132,89 +351,219 @@ NETWORK_NAME: ${NETWORK_NAME}
+	if ! subnets_raw1=$(openstack --os-cloud="${CLOUD}" network show "${NETWORK_NAME}" --format value --column subnets); then
+		log_error "Failed to query subnets for network '${NETWORK_NAME}'"
+		exit 1
+	fi
+	log_info "subnets_raw1=${subnets_raw1}"
 
-if [ -z "${VIP_API}" ] || [ -z "${VIP_INGRESS}" ]
-then
-	echo "Error: VIP_API and VIP_INGRESS must be defined!"
-	exit 1
-fi
+	subnets_raw2=$(echo ${subnets_raw1} | tr "'\"" "\"'")
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)